core/arch/arm/kernel/kern.ld.S

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2014, Linaro Limited
+ * SPDX-License-Identifier: BSD-2-Clause
```

Comment:
The license that follows in this comment could be removed as it only covers the Linaro changes.

The comment with a license below should be left intact though (just as it is now).

Reply:
I agree, but that is a bit more complex to automate. I have to make sure I nuke the license that goes with the Linaro copyright, not the other one(s). That's why I chose to clean only the files that are 100% Linaro copyright.
Maybe I should just take care of this manually.